components/actionmenu/stories/actionmenu.stories.js

```diff
@@ -50,40 +67,227 @@ export default {
-	tags: ["unmigrated"],
+	tags: ["migrated"],
 };
 
 export const Default = ActionMenuGroup.bind({});
 Default.args = {
+	triggerArgs: {
+		iconName: "AddCircle",
+		label: "Add",
+	},
+	menuArgs: {
+		hasActions: true,
+		selectionMode: "multiple",
+		items: [{
+			heading: "Menu section header",
+			description: "Menu section description",
+			items: [{
+				label: "Menu item",
+				iconName: "Circle",
+			},
+			{
+				label: "Menu item",
+				iconName: "Circle",
+			},
+			{
+				label: "Menu item",
+				iconName: "Circle",
+			}],
+		}, {
+			heading: "Menu section header",
+			description: "Menu section description",
+			selectionMode: "none",
+			hasActions: false,
+			items: [{
+				label: "Menu item",
+				iconName: "Circle",
+			},
+			{
+				label: "Menu item",
+				iconName: "Circle",
+			},
+			{
+				label: "Menu item",
+				iconName: "Circle",
+			},],
+		}],
+	},
+};
+
+// ********* DOCS ONLY ********* //
+/**
+ * By default, the menu is opened by pressing the trigger element or activating it via the <kbd>Space</kbd> or <kbd>Enter</kbd> keys. However, there may be cases where the trigger should perform a separate action on press such as selection, and should only display the menu when long pressed. For this use-case, the menu will only be opened when pressing and holding the trigger or by using the <kbd>Option</kbd> (Alt on Windows) + <kbd>Down arrow</kbd>/<kbd>Up arrow</kbd> keys while focusing the trigger.
+ *
+ * This example illustrates the expected visuals and states of the action menu for a trigger with both long press and short press behaviors.
+ *
+ * Please note that the long press functionality is not implemented in this documentation and the example serves only as a visual reference.
+ */
+export const LongPress = Template.bind({});
+LongPress.args = {
+	position: "end-top",
+	hasLongPress: true,
 	isOpen: true,
-	position: "bottom",
-	label: "More actions",
-	iconName: "More",
-	items: [
-		{
-			label: "Action 1",
-		},
-		{
-			label: "Action 2",
-		},
-		{
-			label: "Action 3",
+	triggerArgs: {
+		iconName: "CropRotate",
+	},
+	menuArgs: {
+		customStyles: {
+			"--mod-menu-inline-size": "max-content",
 		},
-		{
-			label: "Action 4",
+		selectionMode: "single",
+		items: [{
+			label: "Crop rotate",
+			iconName: "CropRotate",
+			isSelected: true,
+		}, {
+			label: "Slice",
+			iconName: "VectorDraw",
+		}, {
+			label: "Clone stamp",
+			iconName: "StampClone",
+		}],
+	},
+};
+LongPress.tags = ["!dev"];
+LongPress.parameters = {
+	chromatic: {
+		disableSnapshot: true,
+	},
+};
+
+/**
+ * Action menus can be positioned in four locals relative to the trigger but <em>only one menu can be triggered at a single time</em>.
+ */
+export const PlacementOptions = (args, context) => ArgGrid({
+	Template,
+	argKey: "position",
+	withBorder: false,
+	...args,
+}, context);
+PlacementOptions.args = {
+	triggerArgs: {
+		iconName: "More",
+	},
+	menuArgs: {
+		customStyles: {
+			"--mod-menu-inline-size": "max-content",
```

Comment:
You may have already left a comment somewhere about this, but instead of using the mod here, is it possible to use a `--spectrum` custom property? If we're going to remove mods, would it make sense to not introduce another place where one is being used?

There's a couple more menu mods in the `PlaceholderIcons` args as well.

Reply:
I think removing the mods in some of these more complex components are going to be a bigger project and I didn't want to expand the scope of action menu by addressing that quite yet.